packages/@aws-cdk/core/lib/private/refs.ts

```diff
@@ -156,11 +161,36 @@ function findAllReferences(root: IConstruct) {
 // export/import
 // ------------------------------------------------------------------------------------------------
 
+function createSsmParameter(consumer: Stack, reference: CfnReference): Intrinsic {
+  const exportingStack = Stack.of(reference.target);
+  const id = makeUniqueId([ JSON.stringify(exportingStack.resolve(reference)) ]);
```

Comment:
I can't parse this. What does this ultimately end up as? Do you have an example?

Reply:
In the case, of the lambda layer, this is `RefLayerB20D2F06`

Reply:
Some more examples - https://github.com/aws/aws-cdk/pull/8891/files#diff-11b880f9399e0ddb6ce36b79114cfe99R897 and https://github.com/aws/aws-cdk/commit/dea99668337090239db01638b8f05177f5881393